pyproject.toml

```diff
@@ -104,8 +104,19 @@ version_files = [
     "cookiecutter/target-template/{{cookiecutter.target_id}}/pyproject.toml:singer-sdk",
 ]
 
+[tool.coverage.paths]
+source = [
+    "singer_sdk/",
+    "*/singer_sdk",
+]
+tests = [
+    "tests/",
+    "*/tests",
+]
+
 [tool.coverage.run]
-source = ["singer_sdk"]
+branch = true
```

Comment:
This makes coverage a bit more strict by checking branches in `if` statements. This means our overall coverage will seem to have reduced, but it's only because we're now moving the bar slightly up.

See https://coverage.readthedocs.io/en/6.4.1/branch.html#branch